pyproject.toml

```diff
@@ -10,15 +10,14 @@ version = "1.0.0"
 
 [tool.poetry.dependencies]
 PyYAML = "^5.4.1"
-autogluon = "^0.3.1"
-catch22 = "^0.2.0"
+mxnet = "1.9"
+autogluon = "0.5.0"
 click = "^7.1.2"
 fastparquet = "^0.6.1"
 fbprophet = "^0.7.1"
-gluonts = {git = "https://github.com/awslabs/gluon-ts.git", rev = "7c94c1149875f6ad2e0d7b0a6bcee952f14d3fb1"}
+gluonts = "0.9.5"
```

Comment:
Should we use `0.10`?

I also would not fix the patch version.

Reply:
ok, I will change it.

Reply:
I notice that version 0.10 has removed `save_datasets, save_metadata`,  is there an alternate interface?